js/graviex.js

```diff
@@ -0,0 +1,1012 @@
+                        'order/delete', // Requires access_key, tonce, signature and api secret AND id (unique orderid).
+                    ],
+                },
+            },
+            'fees': {
+            },
+            'limits': {
+                'amount': {
+                    'min': 0.001,
+                    'max': undefined,
+                },
+            },
+            'precision': {
+                'amount': 8,
+                'price': 8,
+            },
+        });
+    }
+
+    async fetchMarkets (params = {}) {
+        const response = await this.publicGetMarkets ();
+        // {
+        //     "id": "giobtc",
+        //     "name": "GIO/BTC",
+        // };
+        const markets = response;
+        const result = [];
+        for (let i = 0; i < markets.length; i++) {
+            const market = markets[i];
+            const id = this.safeString (market, 'id');
+            let symbolParts = this.safeString (market, 'name');
+            symbolParts = symbolParts.split ('/');
+            const baseId = symbolParts[0];
+            const quoteId = symbolParts[1];
+            const base = this.commonCurrencyCode (baseId);
+            const quote = this.commonCurrencyCode (quoteId);
+            const symbol = base + '/' + quote;
+            const active = true;
+            result.push ({
+                'id': id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'active': active,
+                'info': markets[i],
+            });
+        }
+        return result;
+    }
+
+    parseTicker (ticker, market = undefined) {
+        const timestamp = this.milliseconds ();
+        const symbol = market['symbol'];
+        // ticker = ticker['ticker'];
+        const last = this.safeFloat (ticker, 'last');
+        return {
+            'symbol': symbol,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeFloat (ticker, 'high'),
+            'low': this.safeFloat (ticker, 'low'),
+            'bid': this.safeFloat (ticker, 'buy'),
+            'bidVolume': undefined,
+            'ask': this.safeFloat (ticker, 'sell'),
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': undefined,
+            'close': last,
+            'last': last,
+            'previousClose': undefined,
+            'change': this.safeFloat (ticker, 'change'),
+            'percentage': undefined,
+            'average': undefined,
+            'baseVolume': this.safeFloat (ticker, 'volume'),
+            'quoteVolume': this.safeFloat (ticker, 'volume2'),
+            'info': ticker,
+        };
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        await this.loadMarkets ();
```

Comment:
This is not working properly (returns all tickers) because you're sending the market as an url parameter but should be a segment of the path:

Currently: `https://graviex.net//webapi/v3/tickers?market=btcusd`

Correct way: `https://graviex.net//webapi/v3/tickers/btcusd`

You just need to add to the endpoints: 'tickers/{market}'

![image](https://user-images.githubusercontent.com/43336371/153487183-e1368669-f519-474d-a221-e6fd9c3d8dd4.png)

Reply:
Ok. Resolved in latest commit [#11697](https://github.com/ccxt/ccxt/pull/11697/commits/ac050124c3ca15c30f7935008f8bc0e042e9bddd)